core/db/migrate/20150314013438_add_missing_indexes.rb

```diff
@@ -0,0 +1,27 @@
+class AddMissingIndexes < ActiveRecord::Migration
+  def change
+    add_index   :spree_promotion_rules_users, 
+                        [:user_id, :promotion_rule_id], 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Trailing whitespace detected.